engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/interop/syntax/HostValueToEnsoNode.java

```diff
@@ -63,8 +63,8 @@ long doByte(byte i) {
   }
 
   @Specialization
-  long doChar(char i) {
-    return i;
+  Text doChar(char i) {
+    return Text.create(Character.toString(i));
   }
```

Comment:
Nothing against this change, but I'd like to learn the rationale - why are we converting a single character `char` into a `Text` and not keep it as a number?

I guess both approaches seem okay-ish, I guess indeed keeping as Text seems _better_.

But I'm curious mostly because I assume the previous handling was causing some TCK test failure?

Reply:
TCK requires `char` to be seen as single character `String`: